src/bpffeature.cpp

```diff
@@ -739,4 +739,18 @@ bool BPFfeature::has_kernel_dwarf()
   return dwarf->has_debug_info();
 }
 
+bool BPFfeature::has_kernel_func(const std::string& name)
+{
+  if (!has_btf())
```

Comment:
Is it not possible for us to even attempt to call a kfunc if we dont' have BTF?

Reply:
No, b/c we don't know the signature and therefore are not able to construct the call. In addition, kfuncs are "called" by their BTF ID so the kernel where this is run has to have BTF.